apps/class-solid/src/components/plots/LinePlot.tsx

```diff
@@ -32,28 +30,3 @@ function Line(d: ChartData<Point>) {
     </path>
   );
 }
-
-export default function LinePlot({
```

Comment:
This module file name does not cover its contents. We should rename it to Line.tsx/